.github/fabricbot.json

```diff
@@ -0,0 +1,6871 @@
+[
```

Comment:
Why is this file under .github?

Reply:
It's just the convention employed by Fabric Bot

Reply:
Cf. https://microsoft.sharepoint.com/teams/FabricBot/SitePages/Config-as-code.aspx